requirements.txt

```diff
@@ -1,3 +1,10 @@
+certifi==2022.6.15
+charset-normalizer==2.0.12
+idna==3.3
 Pillow==9.0.1
-praw==7.5.0
+praw==7.6.0
+prawcore==2.3.0
 requests==2.27.1
+update-checker==0.18.0
+urllib3==1.26.9
+websocket-client==1.3.3
```

Comment:
are these moduls really neccessary?

Reply:
Yes. I used pip freeze to generate this requirement file